### DISCUSSION TITLE
Save cover image to the audiobook folder

### DISCUSSION THREAD

**Discussion by virtos** [PaulWoitaschek/Voice#2303] [Ideas]
Currently the app looks for a cover image file in the book folder, however saves it in the app data. This is inconsistent, why not save it with the book itself?

**Comment by JamesDBartlett3** [comment]
I support this idea as well! 

I love the feature that lets me choose cover art from the internet to override the often low-rez/outdated/incorrect/missing cover art, and I'd love it even more if the cover art files I've selected were saved alongside their associated audiobook files, so I could more easily sync/migrate my audiobook library between devices.
